test/apache_properties_ssl/kafka_server_jaas.conf

```diff
@@ -0,0 +1,20 @@
+KafkaServer {
+   org.apache.kafka.common.security.plain.PlainLoginModule required
+   username="kafkabroker"
+   password="kafkabroker-secret"
+   user_kafkabroker="kafkabroker-secret"
+   user_kafka-broker-metric-reporter="kafkabroker-metric-reporter-secret"
+   user_client="client-secret";
+};
+
+Server {
+   org.apache.zookeeper.server.auth.DigestLoginModule required
+   user_super="admin-secret"
+   user_kafka="kafka-secret";
+};
+
+Client {
+   org.apache.zookeeper.server.auth.DigestLoginModule required
+   username="kafka"
+   password="kafka-secret";
```

Comment:
Is it the credential of test server in Docker?

Reply:
Test server